test-examples/jdi-light-examples/src/main/java/selenium/seleniumPO/HomePage.java

```diff
@@ -1,41 +1,52 @@
 package selenium.seleniumPO;
 
+import com.epam.jdi.light.driver.WebDriverFactory;
 import com.epam.jdi.light.elements.complex.WebList;
 import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
-import org.openqa.selenium.*;
+import org.openqa.selenium.By;
+import org.openqa.selenium.StaleElementReferenceException;
+import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 
 import java.util.List;
 
-import static com.epam.jdi.light.driver.WebDriverFactory.*;
+import static com.epam.jdi.light.driver.WebDriverFactory.getWebDriverFactory;
 
 public class HomePage {
-    private String url = "https://jdi-testing.github.io/jdi-light/index.html";
+    private final String url = "https://jdi-testing.github.io/jdi-light/index.html";
+    private final WebDriverFactory driverFactory = getWebDriverFactory();
+
+    @FindBy(css = ".sidebar-menu [ui=label]")
+    List<WebElement> navigation;
+    @FindBy(css = ".profile-photo")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5482899)